### DISCUSSION TITLE
Skill Calculator Xp till goal

### DISCUSSION THREAD

**Discussion by cfowle190** [runelite/runelite#14950] [Ideas]
**Is your feature request related to a problem? Please describe.**
yes, whenever I use the skill calculator when working towards a goal, I have to manually subtract the amount of xp of my "current experience" from the "target experience" to know how much XP I will need.

**Describe the solution you'd like**
automatically have it appear underneath the target experience with "experience till goal" or something similar

**Describe alternatives you've considered**
doing it manually/hovering over it

**Additional context**
(https://i.imgur.com/wsKCCIq.png)

**Comment by Nolah2** [comment]
Hovering over the "Total Experience" input already gives you a tooltip with remaining exp required. 

With that in mind, it's already calculating all of the details needed so possibly adding a configuration to the plugin config as something like the following could be done:

Remaining Exp Display Type: 
- Textbox Hover
- Textfield

**Comment by LlemonDuck** [comment]
This is already an option within the XP Tracker plugin config:

![Screenshot_20220429_210355](https://user-images.githubusercontent.com/1868974/166089882-694be1bd-da9c-4f78-bae6-2d8844126aa1.png)

![Screenshot_20220429_210433](https://user-images.githubusercontent.com/1868974/166089896-93fde8af-07c0-49f1-bcf7-0c07bdb40a21.png)

EDIT: I see you're talking about the skill calculator, although this may work in lieu for your use case.

**Comment by Enriath** [comment]
The skill calculator already shows the actions to the goal. You set the goal through the vanilla xp drops system
